### ISSUE TITLE
Duplicate subscriberUpdateAttributes API call

### ISSUE THREAD

**Issue by ValerioCietto** [phpList/phplist-plugin-restapi#70]
The calls `subscriberUpdateAttributesMultiple` and `subscriberUpdateAttributes` do exactly the same thing.

It is possible to eliminate one of them or add a deprecate warning?

**Comment by michield** [comment]
Yes, good point. subscriberUpdateAttributesMultiple was recently added, but we should review and possible remove it again. For now, we can keep it in to avoid breaking things, and mark it deprecated in a little while. Thanks.

**Comment by ValerioCietto** [comment]
Thanks for the info.
